server/controllers/authController.js

```diff
@@ -1,93 +1,61 @@
+import dotenv from 'dotenv';
+import { Pool } from 'pg';
 import User from '../models/User';
 import generateToken from '../helpers/generateToken';
-import Database from '../database/Database';
+
+
+dotenv.config();
+
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URL,
+});
+
 
 const user = new User();
-const database = new Database();
 
 export const signup = async (req, res) => {
-  const {
-    firstName, lastName, email, password,
-  } = req.body;
-  const registeredUsers = await database.retrieveAllUsers();
-  if (registeredUsers) {
-    user.users = registeredUsers;
-    const userAlreadyExists = user.findUser(email);
-    if (userAlreadyExists) {
-      res.status(409).json({
-        status: res.statusCode,
-        error: 'Conflict: User Already exists',
-      });
-    } else {
+  try {
+    const {
+      firstName, lastName, email, password,
+    } = req.body;
+    const createUsersTableReq = `CREATE TABLE IF NOT EXISTS 
+    users(
+      id UUID PRIMARY KEY,
+      firstName VARCHAR(128) NOT NULL,
+      lastName VARCHAR(128) NOT NULL,
+      email VARCHAR(128) UNIQUE NOT NULL,
+      password VARCHAR(128) NOT NULL
+      )`;
```

Comment:
@T2Wil  this should be in the database file

Reply:
Solved!